### DISCUSSION TITLE
Custom keybindings for terminal commands & CMD key

### DISCUSSION THREAD

**Discussion by koyomitan3** [zed-industries/zed#12926] [Feature Requests]
I am using the dev build on Linux it's working great I like it! I just want to remap for example CTRL+5 or F5 to run the command `python3 %` where % is the current file I'm editing so I can easily run it.

I can't seem to get what hotkey CMD is equivalent to for non-MacOS users, it's not the Win key apparently, and I can't bind F5 that does nothing. Any helpers?

**Comment by mauro3** [comment]
I think it is the Mod3, which is usually not mapped to any actual key on Linux.  You can check by running `xmodmap`.  Maybe change `cmd` to `alt` in Zed's keyboard map file.
